kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/thermal/machine/casting.js

```diff
@@ -0,0 +1,20 @@
+onEvent('recipes', (event) => {
+    if (global.isExpertMode == false) {
```

Comment:
Filename is incorrect, and there's no id prefix or recipe name :)

Reply:
Oh, worse than that, this file really was useless. The recipe is already in `induction_smelter` where it belongs.